src/components/Table/HashTimeCell/index.tsx

```diff
@@ -0,0 +1,43 @@
+import classNames from "classnames";
+import Wrapper from "@/components/Table/Wrapper";
+import TimeStamp from "@/components/TimeStamp";
+import IdenticonLink from "@/components/IdenticonLink";
+import Cell from "@/components/Table/Cell";
+
+import styles from "./HashTimeCell.module.scss";
+
+interface Props {
+  hash: string;
+  time: string;
+  identicon?: boolean;
+  index: number;
+  link: string;
+  hashLeft?: number | undefined;
+  hashRight?: number | undefined;
+  className?: string;
+}
+function HashTimeCell({
+  hash,
+  time,
+  identicon,
+  index,
+  link,
+  hashLeft,
+  hashRight,
+  className,
+}: Props) {
+  const shortHash = `${hash.substring(0, hashLeft)}...${hash.substring(
+    hash.length - hashRight!
```

Comment:
Maybe there is a better solution than ! in this case. Maybe it should default to 0 instead?

Reply:
Thanks. Yeah, that's much better! Fixed!